src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageKind.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    public enum RazorLanguageKind
```

Comment:
```suggestion
    internal enum RazorLanguageKind
```

Should make all of the types added in this PR internal as well toensure we don't have any unexpected consumers depending on us 😄